themes/FormalMotoko.tmTheme

```diff
@@ -0,0 +1,636 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>author</key>
+	<string>Martin Kühl</string>
+	<key>comment</key>
+	<string>Based on the Quiet Light theme for Espresso by Ian Beck.</string>
```

Comment:
Original theme taken from https://github.com/microsoft/vscode-textmate/blob/main/test-cases/themes/QuietLight.tmTheme

The only addition so far is `keyword.other.verification.contracts`